monai/transforms/transforms.py

```diff
@@ -441,10 +444,514 @@ def __call__(self, img):
         return data
 
 
-# if __name__ == "__main__":
-#     img = np.array((1, 2, 3, 4)).reshape((1, 2, 2))
-#     rotator = RandRotate90(prob=0.0, max_k=3, axes=(1, 2))
-#     # rotator.set_random_state(1234)
-#     img_result = rotator(img)
-#     print(type(img))
-#     print(img_result)
+class AffineGrid:
+    """
+    Affine transforms on the coordinates.
+    """
+
+    def __init__(self,
+                 rotate_params=None,
+                 shear_params=None,
+                 translate_params=None,
+                 scale_params=None,
+                 as_tensor_output=True,
+                 device=None):
+        self.rotate_params = rotate_params
+        self.shear_params = shear_params
+        self.translate_params = translate_params
+        self.scale_params = scale_params
+
+        self.as_tensor_output = as_tensor_output
+        self.device = device
+
+    def __call__(self, spatial_size=None, grid=None):
```

Comment:
If this class is to **generate grid** instead of **transform image**, why not define it as a utility?

Reply:
i think the user might load "grid" data directly and apply grid transform here

Reply:
OK.
1. It's acceptable.
2. But I am not sure whether we should explicitly define "transforms" only for images.
@ericspod @atbenmurray , do you guys have any comments?
Thanks.

Reply:
perhaps that's not needed, e.g. we allow the users to apply "seg_transforms" on classification labels as well https://github.com/Project-MONAI/MONAI/blob/master/monai/data/nifti_reader.py#L113, I think it's a nice feature for flexibilities